services/SecurityService.go

```diff
@@ -0,0 +1,92 @@
+package services
+
+import (
+	"fmt"
+	"github.com/jasongauvin/wikiPattern/models"
+	"golang.org/x/crypto/bcrypt"
+	"time"
+)
+
+type RegisterForm struct {
+	Email    string `form:"userEmail" binding:"required"`
+	Password string `form:"userPassword" binding:"required"`
+}
+
+type LoginForm struct {
+	Email    string `form:"userEmail" binding:"required"`
+	Password string `form:"userPassword" binding:"required"`
+}
+
+// SaveUser creates a user.
+// Return the created object
+func SaveUser(email string, password string) (*models.User, error) {
```

Comment:
If the whole point of this function is to just create a struct and set the email and password, let the called create the struct with the correct field values and directly call the model, so this method is not necessary at all